<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogParameters</class>
 <widget class="QDialog" name="DialogParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>20</y>
     <width>29</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sex:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Sex_comboBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>70</y>
     <width>30</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AgeLine">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>120</y>
     <width>83</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height (cm):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HeightLine">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>140</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weight (kg):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WeightLine">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lifestyle:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Activity_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
   <item>
    <property name="text">
     <string>Moderate activity 
(sedentary work and light physical
exercises or classes 1-3 times a week; housewives)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average activity
(sedentary work plus exercises 3-5 
times per week; housewife with a garden; 
work on legs up to 12 hours without physical work)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>High activity
(intensive exercices 6-7 times per week; 
work on the legs and intense sports)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Very high activity
(athletes and people who have 
hard exercices 6-7 times a week)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="goal_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Lose weight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maintain weight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gain weight</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Main goal:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
